### PR TITLE
docs(oidc): Connectors warning regarding OIDC

### PR DESCRIPTION
**Pull request by sbuettner**
## Description

Mention the current limitation of Connectors not working OIDC.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by sbuettner** [comment]
@christinaausley Feel free to merge this one. I am on FTO as of tomorrow.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md [hunk 1]
```diff
@@ -96,7 +96,9 @@ global:
 For authentication, the Camunda components use the scopes `email`, `openid`, and `offline_access`.
 
 :::note
-Web Modeler does not yet support authentication with a generic OIDC provider.
+Web Modeler and the Connectors do not yet support authentication with a generic OIDC provider.
+
+When using OIDC, please disable Connectors in your deployment (see https://docs.camunda.io/docs/next/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider/).
```
Comment (MaxTru) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md:101]:
```suggestion
When using OIDC, please disable Connectors in your deployment (see https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/values.yaml#L2572).
```

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md [hunk 2]
```diff
@@ -177,6 +179,10 @@ process.
 To successfully authenticate wth Entra ID, you should use the `v2.0` API. This means that
 the `CAMUNDA_IDENTITY_ISSUER_BACKEND_URL` value should end with `/v2.0`.
 
+:::note
+Connectors do not yet support authentication with Microsoft Entra ID as the OIDC provider. When using OIDC, please disable Connectors in your deployment (see https://docs.camunda.io/docs/next/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider/).
```
Comment (MaxTru) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md:183]:
```suggestion
Connectors do not yet support authentication with Microsoft Entra ID as the OIDC provider. When using OIDC, please disable Connectors in your deployment (see https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/values.yaml#L2572).
```

**Review by MaxTru — CHANGES_REQUESTED** [review]
Isn't the link wrong @sbuettner

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md (hunk 1 above)
Comment (christinaausley) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md:101]: Or https://docs.camunda.io/docs/next/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd/?

Comment (christinaausley) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md:101]: Regardless, let's make sure to use `[anchor text like this for links]()`.

Comment (MaxTru) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md:101]: Let's use https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/values.yaml#L2572 please.

@christinaausley can you please change the PR? (I am not at my laptop anymore)

Comment (christinaausley) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md:101]: Please get some sleep @MaxTru 😄 I'll get this adjusted.

**Review by christinaausley — APPROVED** [review]

**Review by MaxTru — APPROVED** [review]

**Pull request merged**